backend/api/tests.py

```diff
@@ -70,3 +70,13 @@ def test_post_create_view(self):
         self.assertContains(
             response, self.relative_profile_url, status_code=status.HTTP_201_CREATED
         )
+
+    def test_root_status_code(self):
+        """
+        Test APIRootView for status code 200.
+        """
+        url = reverse("api:api-root")
+        response = self.client.options(url)
```

Comment:
Let's change this to a get. The test should really test that the View exists at the location we expect it to be in. The error you were getting was because the test runner doesn't add any host headers so let's add a default one:


```python

response = self.client.get(url, HTTP_HOST="example.com")
```

Adding the HTTP header like that should fix the test.